www/js/contact.js

```diff
@@ -1,3 +1,11 @@
+$$(document).on('page:beforein', '.page[data-name="contact"]', function() {
```

Comment:
Could these two event handlers be combined into one selector somehow?
```suggestion
$$(document).on('page:beforein', '.page[data-name="contact"], .page[data-name="smart-select-page"]', function() {
```